.github/workflows/deploy.yaml

```diff
@@ -55,7 +55,7 @@ jobs:
           tags: ${{ steps.meta_migration.outputs.tags }}
           labels: ${{ steps.meta_migration.outputs.labels }}
 
-      - uses: cowprotocol/autodeploy-action@v1
+      - uses: cowprotocol/autodeploy-action@v2
```

Comment:
Where did the `autodeploy-action` version actually get bumped such that this change is necessary (and would work)?

Reply:
https://github.com/cowprotocol/autodeploy-action/releases/tag/v2